resources/tools/INSTALL.txt

```diff
@@ -0,0 +1,46 @@
+                                                    ~ BALLERINA ~
+                                                    =============
+
+
+PREREQUISITES
+-------------
+
+Ballerina requires JRE 8 Update 172+.
+It is recommended to use AdoptOpenJDK. Check https://adoptopenjdk.net/releases.html to download the JRE.
+
+
+INSTALLING BALLERINA
+--------------------
+
+- Unzip the Ballerina archive in the desired directory.
+
+- Set the path of the installation directory to the BALLERINA_HOME environment variable.
+
+- Add the BALLERINA_HOME environment variable to your system's path variable.
+
+- If you have installed Java 8 already, you are all set! Just make sure that the JAVA_HOME environment variable is set correctly.
+
+- If you do not have Java 8, or if you have a different version of Java, download a compatible JRE 8 version as specified in the prerequisites.
+	- To set the obtained JRE as the one used by Ballerina, extract the downloaded JRE to the <BALLERINA_HOME>/dependencies/jdk8u202-b08-jre/ directory.
+
```

Comment:
Once the jre is copied to this directory, the final directory structure would look like

```
$ tree <BALLERINA_HOME>/dependencies/jdk8u202-b08-jre -L 1
/Library/Ballerina/dependencies/jdk8u202-b08-jre
├── ASSEMBLY_EXCEPTION
├── LICENSE
├── THIRD_PARTY_README
├── bin
├── lib
├── man
└── release

3 directories, 4 files

```